build.override.targets

```diff
@@ -9,6 +9,13 @@
     </TraversalBuildDependsOn>
   </PropertyGroup>
   
+    <PropertyGroup>
+    <TraversalCleanDependsOn>
+      WcfClean;
```

Comment:
I can't find a way to make this work as a single target.  I would like to keep them separate targets for now.  Details: having one target depend on another makes that target get skipped if the other target ran first.  Hence, 'build /t:Clean;BuildAndTest' executs the WcfClean once only.